docs/modules/sources/pages/csv.adoc

```diff
@@ -0,0 +1,40 @@
+= CSV Source
+
+The `xls` source can also create a requisition from a UTF-8 encoded comma separated variable (CSV) file if the given file name has the `.csv` suffix.
+
+In this case the first line of the csv file should contain a comma separated list of headers in the same way the spreadsheet should be configured in a normal xls source. 
+Alternatively, a property can be set containing comma separated set of headers to be is used as the first line of the csv file.
+This allows csv file to be raad which don't have a header line.
+
+[options="header",autowidth"]
+|===
+| Parameter         | Required | Description
+| `source`          | *        | set `xls` to use the XLS source for this requisition
```

Comment:
```suggestion
| `source`          | *        | Set `xls` to use the XLS source for this requisition.
```
We are formatting tables differently in the main docs now, but no point in updating it here until we're ready to update the entire PRIS documentation.